target/xilinx/constraints/vcu118.xdc

```diff
@@ -0,0 +1,199 @@
+# Copyright 2022 ETH Zurich and University of Bologna.
+# Solderpad Hardware License, Version 0.51, see LICENSE for details.
+# SPDX-License-Identifier: SHL-0.51
+#
+# Nicole Narr <narrn@student.ethz.ch>
+# Christopher Reinwardt <creinwar@student.ethz.ch>
+# Cyril Koenig <cykoenig@iis.ee.ethz.ch>
+# Paul Scheffler <paulsc@iis.ee.ethz.ch>
+
+#############
+# Sys Clock #
+#############
+
+# 125 MHz input clock
+set_property -dict {LOC AY24 IOSTANDARD LVDS} [get_ports sys_clk_p]
```

Comment:
Is it possible to move this in the "Assign Pins" section?

Reply:
Yes, I'll fix it.